docs-src/docs/kognic-io/pre_annotations.md

```diff
@@ -23,12 +23,12 @@ see [Creating Multiple Inputs With One Call](working_with_scenes_and_inputs.md#c
 ### 1. Creating a scene
 
 :::note
-The scene is a subset of the input, specifically referring to the data, such as images or point clouds.
-An input is what is created when this data is ready to be annotated.
+The scene is a subset of the input, specifically referring to the raw data, such as images or point clouds.
+An input is created when this data is associated to a request in a project with task definition and guildline, and is ready to be annotated.
```

Comment:
I think that this note can be removed completely. It's not correct to say that _a scene is a subset of an input_ and it doesn't really add much information other than that

Reply:
Done